pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -205,26 +205,31 @@ func (this *SnapshotManager) DeleteSnapshot(peID astrolabe.ProtectedEntityID) er
 	return nil
 }
 
-func (this *SnapshotManager) DeleteProtectedEntitySnapshot(peID astrolabe.ProtectedEntityID, toRemote bool) error {
-	this.Infof("SnapshotManager.deleteProtectedEntitySnapshot Called")
-	this.Infof("Input arguemnts: peID = %s, toRemote = %v", peID.String(), toRemote)
+func (this *SnapshotManager) DeleteLocalProtectedEntitySnapshot(peID astrolabe.ProtectedEntityID) error {
+	this.Infof("SnapshotManager.deleteLocalProtectedEntitySnapshot Called")
+	return this.deleteProtectedEntitySnapshot(peID, this.ivdPETM)
+}
+
+func (this *SnapshotManager) DeleteRemoteProtectedEntitySnapshot(peID astrolabe.ProtectedEntityID) error {
+	this.Infof("SnapshotManager.deleteRemoteProtectedEntitySnapshot Called")
+	return this.deleteProtectedEntitySnapshot(peID, this.s3PETM)
+}
+
+func (this *SnapshotManager) deleteProtectedEntitySnapshot(peID astrolabe.ProtectedEntityID, petm astrolabe.ProtectedEntityTypeManager) error {
```

Comment:
How about deleteSnapshotFromRepo for this instead?  I'm confused why deleteProtectedEntitySnapshot is different from DeleteSnapshot and DeleteSnapshot actually winds up calling this but via DeleteLocalSnapshot and DeleteRemoteSnapshot